pkg/io/video/throttle.go

```diff
@@ -0,0 +1,27 @@
+package video
+
+import (
+	"image"
+	"time"
+)
+
+// Scale returns video throttling transform.
```

Comment:
knit: could you also add some comments about what rate is? I think, from my understanding, rate is frame per second. Am I right?

Reply:
That's right.
I fixed the comment.